docs/openstack/glance_adoption.md

```diff
@@ -74,7 +74,7 @@ spec:
     enabled: true
     template:
       databaseInstance: openstack
-      containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
+      containerImage: {{ container_registry|default("quay.io") }}/{{ container_namespace|default("podified-antelope-centos9") }}/openstack-glance-api:{{ container_tag | default("current-podified") }}
```

Comment:
Ditto here, we can't use jinja in docs. So this issue was about automation (test suite) mainly. I think it got fixed here but we forgot to close the issue: https://github.com/openstack-k8s-operators/data-plane-adoption/pull/148/files

Reply:
Or actually, we may be able to remove the hardcoded images from docs by removing the `containerImage` lines completely. I think the operators should be defaulting to the correct images. I think that's what i had in mind when i reported the issue. I'm not sure if that approach will work for the EDPM doc too though, i don't know if the variables are defaulted in EDPM Ansible.